airgun/views/contentview.py

```diff
@@ -299,6 +300,26 @@ def search(self, query, repo=None):
             self.searchbox.search(query)
             return self.table.read()
 
+    @View.nested
+    class module_streams(SatSecondaryTab):
```

Comment:
it seems that whole content of previous class is just copied, so maybe we can do something like:
```
    class module_streams(rpm_packages):
        TAB_NAME = 'Module Streams'
```

or create some generic class for that purpose:
```
    class generic_class_name_abc(SatSecondaryTab)
    .......

    @View.nested
    class rpm_packages(generic_class_name_abc):
        TAB_NAME = 'rpm Packages'

    @View.nested
    class module_streams(generic_class_name_abc):
        TAB_NAME = 'Module Streams'

Reply:
Let me try that

Reply:
Added new generic class name as EntitySearchView and inherited in both package and module stream classes